components/docs/VersionSelect.jsx

```diff
@@ -18,9 +17,11 @@ export default function VersionSelect({
 }) {
   const [selectedVersion, setSelectedVersion] = useState(version)
 
-  versions = versions.sort(function(first, second){
-	  return compareVersions(labelFromVersion(first), labelFromVersion(second))
-  }).reverse()
+  versions = versions
+    .sort(function (first, second) {
+      return compareVersions(labelFromVersion(first), labelFromVersion(second))
+    })
+    .reverse()
```

Comment:
This is the formatting applied by `prettier` using the `.prettierrc` file which already exists in this repo.